audiorecorder/src/test/java/org/odk/collect/audiorecorder/recording/AudioRecorderViewModelTest.kt

```diff
@@ -153,4 +154,16 @@ abstract class AudioRecorderViewModelTest {
         runBackground()
         assertThat(session.value?.failedToStart, equalTo(null))
     }
+
+    @Test
+    fun supportsSerializableSessionIds() {
```

Comment:
Annoyingly we need a test to drive out some of the changes here as `String` is `Serializable`